tests/chainer_tests/links_tests/loss_tests/test_hierarchical_softmax.py

```diff
@@ -107,5 +107,21 @@ def test_to_cpu(self):
         self.assertTrue((f.paths == g.paths).all())
         self.assertTrue((f.codes == g.codes).all())
 
+    def _test_sampling(self, x):
+        x = chainer.Variable(x)
+        result = self.link.sampling(x)
+        self.assertEqual(len(result), x.shape[0])
+        for word_id in result:
+            self.assertEqual(type(word_id), int)
```

Comment:
Thank you. But I think we should use `assertLessEqual` because there is not `assertLe` method in unit python test. I will use it!